packages/store/src/internal/bootstrapper.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+
+import { ReplaySubject, Observable } from 'rxjs';
+
+@Injectable()
+export class Bootstrapper {
+  /**
+   * Use `ReplaySubject`, thus we can get cached value even if the stream is completed
+   */
+  private bootstrap$ = new ReplaySubject<boolean>(1);
```

Comment:
why (1)?

Reply:
@splincode We have to cache 1 value - `true`. Even if this stream is completed - any subscriber will get `true` value if application is bootstrapped. If you don't provide `bufferSize` argument - the default value is `Number.POSITIVE_INFINITY`.

Reply:
add comments into code

Reply:
@splincode where? :wink: